tests/BenchmarkDotNet.Tests/Mocks/MockFactory.cs

```diff
@@ -24,9 +24,10 @@ namespace BenchmarkDotNet.Tests.Mocks
 {
     public static class MockFactory
     {
-        public static Summary CreateSummary(Type benchmarkType)
+        public static Summary CreateSummary(Type benchmarkType, bool hideColumnStdDev = false)
```

Comment:
I think that's a bit too specific for the factory.
```suggestion
        public static Summary CreateSummary(Type benchmarkType, params IColumnHidingRule[] columHidingRules)
```